www/_posts/2016-06-03-cordova-windows-4.4.0.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+author:
+    name: Vladimir Kotikov
+    url: https://github.com/vladimir-kotikov
+title:  "Apache Cordova Windows 4.4.0"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `Cordova Windows 4.4.0` has been released!
+
+This release adds some significant improvements, such as increased 'prepare' performance due to new 'incremental prepare' feature,
+embedded splashscreen plugin support and others. See below for full list of changes.
+
+The next Cordova CLI version will automatically start using this version of **Cordova-Windows** when creating new projects.
+
+To upgrade:
+
+    npm install -g cordova
+    cd my_project
+    cordova platform update windows@4.4.0
+
+To add it explicitly:
+
+    cordova platform add windows@4.4.0
+
+For non-CLI projects or for pre-3.0 projects, refer to the [upgrade guide](http://cordova.apache.org/docs/en/dev/guide/platforms/win8/upgrade.html).
```

Comment:
I think we should stop referencing this page in our release notes - it's not as relevant.